lib/formatter.js

```diff
@@ -319,10 +319,25 @@ function formatOrder(name, traffickerId, partner) {
   };
 }
 
+/**
+ * Generate price points.
+ *
+ * @return {Array}              Price points.
+ */
+var generatePricePoints = function() {
+  var points = [];
+  points.push("0.00");
```

Comment:
Maybe should be removed, see Juan's comment in the tests about starting with 0.01